CODE_OF_CONDUCT.md

```diff
@@ -2,7 +2,7 @@
 
 ## Our Pledge
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

Comment:
looks good to avoid potential issues. BTW, how about we try to newly write **NN Streamer CoC**  rather than we update the contents based on the existing CoC document.